src/components/xy_chart/series/vertical_bar_series.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { VerticalBarSeries } from 'react-vis';
+
+export class EuiVerticalBarSeries extends VerticalBarSeries {
+
+  render() {
+    const { name, data, color, onClick, ...rest } = this.props;
+    const isHighDataVolume = data.length > 80 ? true : false;
+
+    return (
+      <VerticalBarSeries
+        key={name}
+        onSeriesClick={onClick}
+        color={color}
+        style={{
+          strokeWidth: isHighDataVolume ? 0.25 : 1,
+          stroke: 'white',
+          rx: isHighDataVolume ? 0.5 : 2,
+          ry: isHighDataVolume ? 0.5 : 2,
+        }}
+        data={data}
+        {...rest}
+      />
+    );
+  }
+}
+
+EuiVerticalBarSeries.displayName = 'EuiVerticalBarSeries';
+
+EuiVerticalBarSeries.propTypes = {
+  /** The name used to define the data in tooltips and ledgends */
+  name: PropTypes.string.isRequired,
+  /** Array<{x: string|number, y: number}> */
+  data: PropTypes.arrayOf(PropTypes.shape({
+    x: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.number
+    ]),
+    y: PropTypes.number,
+  })).isRequired,
+  /** Without a color set, a random EUI color palette color will be chosen */
```

Comment:
doesn't look like color defaults to one from EUI palette

Reply:
The XYChart, before rendering the series, it assure if a color exist on the props, if not it assign a color from the `VISUALIZATION_COLORS` scale.
I need to work a bit more on colors since this was only touched by @mattapperson and there are various use cases for use different coloring

Reply:
@markov00 You should speak to @snide about this as the choice to only support EUI color palette was made to limit the colors to safe colors for those with visual impairment.

Reply:
@mattapperson Just talked with @snide I will try to force the use of EUI color palette as much as possible on the chart implementation. I will pair with Dave next week so we can also fix some styling together